src/Core/Scheduler/Scheduler.js

```diff
@@ -56,6 +56,8 @@ function _instanciateQueue() {
                 this.counters.executing--;
                 cmd.reject(err);
                 this.counters.failed++;
+                // eslint-disable-next-line no-console
+                console.error(err.stack);
```

Comment:
Please do directly `console.error(err)`. The stack will be displayed, but we'll also have the error message. Also, I'd prefer to have this LOGGER only in debug mode, as user can attach to errors with cmd.promise.

Reply:
ok, how can i know I'm in debug mode in the code ?

Reply:
it's done

Reply:
> Also, I'd prefer to have this LOGGER only in debug mode, as user can attach to errors with cmd.promise.

The cmmand are issued by itowns (eg. `FeatureProcessing`) so an external user can't attach to these promises.
What about logging the N first errors for a given layer?
(eventually adding an option to layers to opt-out from this behavior ?)

Reply:
Good idea to avoid being spam by errors !
I update the code. The new behavior is if you see two same errors, it's means many errors occurs.

Reply:
> What about logging the N first errors for a given layer?
Ok, I show only two errors.

> (eventually adding an option to layers to opt-out from this behavior ?)
As errors are only showed in Debug mode, I think we don't have to do it now.

Is it ok for you ?

Reply:
Yep, thanks.